### PR TITLE
DEV-804: Change DidNamespaceKey to `did-namespace:`

### PR DESCRIPTION
**Pull request by Toktar**
TODO: tests!!!

### CODE REVIEW

**Comment by ankurdotb** [comment]
Task linked: [DEV-705 [performance] Split validation into static and dynamic](https://app.clickup.com/t/6600954/DEV-705)

**Comment by ankurdotb** [comment]
Task linked: [DEV-804 Migrate `did-namespace` key value from `testnettestnet`](https://app.clickup.com/t/6600954/DEV-804)

File: app/app.go [hunk 1]
```diff
@@ -317,6 +317,14 @@ func New(
 		return initialVM, nil
 	})
 
+	app.UpgradeKeeper.SetUpgradeHandler("v0.5", func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		ctx.Logger().Info("Handler for upgrade plan: v0.5")
+
+		app.Migration05(ctx)
```
Comment (askolesov) [inline app/app.go:323]: I suggest using the approach from this guide:
https://docs.cosmos.network/master/building-modules/upgrade.html

**Review by askolesov — CHANGES_REQUESTED** [review]

File: app/app.go (hunk 1 above)
Comment (ankurdotb) [reply app/app.go:323]: Yes, agreed and like I mentioned before!

**Review by ankurdotb — COMMENTED** [review]
Review whether linting failures should be resolved?

**Review by askolesov — DISMISSED** [review]

**Review by ankurdotb — APPROVED** [review]

**Pull request merged**